src/controllers/HierarchicalFacetSearchController.ts

```diff
@@ -1,28 +1,53 @@
 import { IDynamicHierarchicalFacet } from '../ui/DynamicHierarchicalFacet/IDynamicHierarchicalFacet';
 import { FacetSearchType, IFacetSearchRequest } from '../rest/Facet/FacetSearchRequest';
-import { IFacetSearchResponse } from '../rest/Facet/FacetSearchResponse';
 import { flatten } from 'underscore';
+import { IFacetSearchResponse } from '../rest/Facet/FacetSearchResponse';
 
 export class HierarchicalFacetSearchController {
+  private page = 1;
```

Comment:
Is `page` meant to be a count? If so, I feel like a name such as `pages` or `pageCount` would be a little better. If it's meant to be an index, could it start at `0`?

Reply:
PageCount is alright, i use it directly as a multiplier hence the 1